elasticsearch/src/test/java/com/tcdi/zombodb/query_parser/TestQueryRewriter.java

```diff
@@ -4058,6 +4102,7 @@ public void testStopWordRemoval_allField() throws Exception {
         );
     }
 
+    @Ignore
```

Comment:
I'd like to suggest that as you're going through this process you don't tag tests as `@Ignore`.  

My concern is that tests tagged as such will get overlooked as you make progress, and for what it's worth, these things don't exist because writing tests are fun.  :)  They exist to validate specific behaviors.

If you're just trying to get past the build process, you can easily compile with a `-DskipTests` argument to Maven.

I really believe we'd be better served if you reverted these `@Ignore` tags.